client/src/components/button/button.sass

```diff
@@ -1,7 +1,13 @@
+@use "../../../_library"
```

Comment:
I like what you're doing with this, but I think you should break it down further.
`colors`, `layout`, `mixins` maybe?

Could totally leave it for now, its just nice to know exactly where to look for things

Reply:
I think when it gets big enough to be harder to look around we split. I know that's probably the beginner way of looking at it, so feel free to enforce splitting now. 

At this point I don't what we would want to split on (your suggestions probably if we did split now), so I normally go by "add it all in the same file until I realize it's getting messy"

Reply:
also, are you okay with using the _library notation? I figured it's probably only going to be for this project so calling it private would be okay. But I'm not tied to it if you think it shouldn't be.

Reply:
`_partial` is standard sass notation, so 👍

Reply:
As for what to split on, whatever makes organizational sense. You kinda get a feel for it eventually